R/combined_seasonal_output.R

```diff
@@ -0,0 +1,137 @@
+#' Compute seasonal onset and burden levels from seasonal time series observations.
+#'
+#' @description
+#'
+#' This function performs automated and early detection of seasonal epidemic onsets and calculates the burden
+#' levels from time series dataset stratified by season. The seasonal onset estimates growth rates for consecutive
+#' time intervals and calculates the sum of cases. The burden levels use the previous seasons to calculate the levels
+#' of the current season.
+#' @inheritParams seasonal_burden_levels
+#' @inheritParams seasonal_onset
+#' @param tsd `r rd_tsd()`
```

Comment:
Continued from above:

If you make "rd_tsd()" to a character string you would need to update here as well
```suggestion
#' @param tsd `r rd_tsd`
```